administration/src/bp-modules/self-service/hooks/useCardGeneratorSelfService.ts

```diff
@@ -31,12 +33,46 @@ type UseCardGeneratorSelfServiceReturn = {
   downloadPdf: (code: CreateCardsResult, fileName: string) => Promise<void>
 }
 
+const parseDate = (dateString: string): PlainDate | undefined => {
+  try {
+    return PlainDate.from(dateString)
+  } catch {
+    return undefined
```

Comment:
haven't included this part yet 
> Please also provide some Error handling if birthday can not be parsed and show a toast to the user.
because if I add a toast here, it's displayed twice. have to think a while how to avoid that

Reply:
but actually I'm not sure if that's really needed. the users are not supposed to pass query params, and if it happens for some reason through the app - there will be a validation error for an empty field anyway

Reply:
Yes you are probably right. I came to the same result during the birthday extension improvement task